apisupport/maven.apisupport/src/org/netbeans/modules/maven/apisupport/NbmWizardIterator.java

```diff
@@ -62,12 +64,12 @@
     static {
         NB_MODULE_ARCH = new Archetype();
         NB_MODULE_ARCH.setGroupId("org.apache.netbeans.archetypes"); //NOI18N
-        NB_MODULE_ARCH.setVersion("1.17"); //NOI18N
+        NB_MODULE_ARCH.setVersion("1.18"); //NOI18N
         NB_MODULE_ARCH.setArtifactId("nbm-archetype"); //NOI18N
 
         NB_APP_ARCH = new Archetype();
         NB_APP_ARCH.setGroupId("org.apache.netbeans.archetypes"); //NOI18N
-        NB_APP_ARCH.setVersion("1.22"); //NOI18N
+        NB_APP_ARCH.setVersion("1.23"); //NOI18N
         NB_APP_ARCH.setArtifactId("netbeans-platform-app-archetype"); //NOI18N
```

Comment:
First @oyarzun I'm sorry the related issue is to another JIRA now. NETBEANSINFRA-6
Maybe revert the version to released. So we can integrate your changes not hurting.

I'll try to get the plugins archetype with your modification merge done asap and we can see if new version are found.

Reply:
@ebarboni I saw that you merged apache/netbeans-mavenutils-archetype-nbm-archetype#1 and apache/netbeans-mavenutils-archetype-netbeans-platform-app-archetype#2 as part of the fix for [NETBEANSINFRA-6][1].

So you would like me to revert the archetype version numbers in the diff you mention in your previous comment? 

My thought was that if for some reason the code to query the maven repo for the latest versions did not return the latest value then it would use the defaults which has the fix for [NETBEANSINFRA-6][1]. Although if querying the repo fails then the user probably has other problems.

[1]: https://issues.apache.org/jira/browse/NETBEANSINFRA-6

Reply:
Reverting the version is just to make this PR works, archetype need to be released via voting first.
Making possible to test without needing archetype.
in short this PR is for get latest archetype.

Once merged will be a second PR only with version changes that can wait for release.

2 steps fix